tests/auth_api/oidc/test_login.py

```diff
@@ -95,21 +99,21 @@ def test__omit_parameter_return_url__should_return_status_400(
 
         # -- Assert ----------------------------------------------------------
 
-        assert r.status_code == 400
+        assert res.status_code == 400
 
     @pytest.mark.unittest
     def test__omit_parameter_fe_url__should_return_status_400(
             self,
             client: FlaskClient,
     ):
         """
-        Omitting the 'fe_url' parameter should result in the endpoint
+        Omitting the 'fe_url' parameter should result in the endpoint.
```

Comment:
```suggestion
        Omitting the 'fe_url' parameter should result in the endpoint.

```
Newline between summary and extended summary.